src/CardContent/Infographic/Infographic.tsx

```diff
@@ -43,10 +50,13 @@ const Infographic = ({ media, version }: InfographicProps) => {
         width: 368
       }}
     >
+      {showSkeleton && <Skeleton width="100%" />}
       <CardMedia
         component="img"
-        image={isVisible ? media : undefined}
-        loading="lazy"
+        alt={hasError ? "card-infographic" : ""}
+        src={isVisible ? media : undefined}
+        onLoad={() => setIsLoaded(true)}
+        onError={() => setHasError(true)}
```

Comment:
Handlers in case an image finished loading or encounter an error, needed for the skeleton